services/director-v2/openapi.json

```diff
@@ -2055,9 +2126,32 @@
           "wallet_info": {
             "wallet_id": 1,
             "wallet_name": "My Wallet"
+          },
+          "pricing_info": {
```

Comment:
Just wondering how this got here?  I think we are not using these fields in the openapi specs/endpoints.

Reply:
Not sure to tell the truth. But I've updated the specs of the director-v2